README.md

```diff
@@ -0,0 +1,108 @@
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <!--
+  <a href="https://github.com//majkrzak/kot">
+    <img src="images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+  -->
+
+  <h3 align="center">Kot</h3>
+
+  <p align="center">
+    Packing tool made simple
+    <br />
+    <br />
+    <a href="https://github.com/majkrzak/kot/issues">Report Bug</a>
+  </p>
+</p>
+
+
+
+<!-- TABLE OF CONTENTS -->
+## Table of Contents
+
+* [About the Project](#about-the-project)
+* [Getting Started](#getting-started)
+  * [Installation](#installation)
+* [Usage](#usage)
+* [Contributing](#contributing)
+* [License](#license)
+
+
+
+## About The Project
+
+Project aims for simpler packing than Gradle is currently doing. 
+
+## Getting Started
+
+This is example how you can get started with this tool.
+
+Make sure you have python installed.
+
+```sh
+sudo apt install python3 pip -yqq
+```
+
+### Installation
+
+Clone and install with pip.
+
+
+## Usage
+
+Create example configuration, like bellow and run the script.
+
+```json
+{
+  "name": "my_project",
+  "dependencies": [
+    "org.jetbrains.kotlin:kotlin-stdlib:1.3.50",
+    "org.jetbrains.kotlin:kotlin-reflect:1.3.50",
+    "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.2",
+    "org.koin:koin-core:2.0.1"
+  ],
+  "modules": {
+    "simple":[],
+    "complex": [
+        "simple"
+    ]
+  }
+}
+```
+
```

Comment:
Only currently avaliaveble command is to gnerate ide aconfig
`python -m kot my_project.json`